src/main.js

```diff
@@ -2,42 +2,52 @@ import {
   getMenuWrappedMarkup,
   getSearchMarkup,
   getFilterWrappedMarkup,
+  checkFilterOverdue,
+  checkFilterToday,
+  checkFilterFavorite,
+  checkFilterArchived,
+  checkFilterRepeating,
+  checkFilterTags,
   getEditFormMarkup,
   getBoardMarkup,
   getSortingMarkup,
   getLoadButtonMarkup,
   getTaskCardMarkup,
+  taskList,
+  TASK_BOARD_SIZE
 } from './components';
 
+let displayedTasks = taskList.slice(0, TASK_BOARD_SIZE);
 const menuElements = [
   {name: `new-task`},
   {name: `task`, isChecked: true},
   {name: `statistic`},
 ];
 
+const getTaskCount = (tasks, cb) => tasks.filter(cb).length;
+
+
 const filterElements = [
-  {name: `All`, count: 13, isChecked: true},
-  {name: `Overdue`, count: 0},
-  {name: `Today`, count: 0},
-  {name: `Favorites`, count: 1},
-  {name: `Repeating`, count: 1},
-  {name: `Tags`, count: 1},
-  {name: `Archive`, count: 115},
+  {title: `All`, count: taskList.length, isChecked: true},
+  {title: `Overdue`, count: getTaskCount(taskList, checkFilterOverdue)},
+  {title: `Today`, count: getTaskCount(taskList, checkFilterToday)},
+  {title: `Favorites`, count: getTaskCount(taskList, checkFilterFavorite)},
+  {title: `Repeating`, count: getTaskCount(taskList, checkFilterRepeating)},
+  {title: `Tags`, count: getTaskCount(taskList, checkFilterTags)},
+  {title: `Archive`, count: getTaskCount(taskList, checkFilterArchived)},
 ];
 
-const taskCard = {
-  text: `Example default task with default color.`,
-  date: Date.now(),
-  tags: [`todo`, `personal`, `important`],
-};
-
 const menuContainer = document.querySelector(`.main__control`);
 const mainContainer = document.querySelector(`.main`);
 
 const renderComponent = (container, component, position) => {
```

Comment:
https://github.com/htmlacademy-ecmascript/926049-taskmanager-9/blob/3de2df8feb7da102d938982e37f32f710960baa7/src/main.js#L51-L54
Добавь `beforeend` как параметр по умолчанию:
```suggestion
const renderComponent = (container, component, position = `beforeend`) => {
```